libs/core/src/lib/list/list.component.scss

```diff
@@ -376,16 +376,13 @@
 }
 .fd-list--dropdown .fd-list__item {
     height: auto;
-    padding: .75rem 1rem
+    padding: .75rem 1rem;
```

Comment:
These style changes are an update from fundamental-styles?

Reply:
> I don't think this is acceptable that MultiInput behaves differently based on cozy/compact mode. I think we should take care of it in separate ticket #2948.

@salarenko this is directly from the interaction specification, I've had a few conversations about it with the designers and this is what they want